setup/pkg/fieldlabs/environment_create.go

```diff
@@ -491,14 +510,105 @@ func (e *EnvironmentManager) isNotFound(err error) bool {
 	return strings.Contains(err.Error(), "App not found")
 }
 
-func (e *EnvironmentManager) inviteUsers(envs []Environment) error {
+func (e *EnvironmentManager) getPolicies() (map[string]string, error) {
+	req, err := http.NewRequest(
+		"GET",
+		fmt.Sprintf("%s/v1/policies", e.Params.IDOrigin),
+		nil,
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "build policies request")
+	}
+	req.Header.Set("Authorization", e.Params.SessionToken)
+	req.Header.Set("Accept", "application/json")
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, errors.Wrap(err, "send policies request")
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		panic(err.Error())
+	}
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf("GET /v1/policies %d: %s", resp.StatusCode, body)
+	}
+	var policies []PolicyListItem
+	err = json.Unmarshal([]byte(body), &policies)
+	if err != nil {
+		return nil, errors.Wrap(err, "list policies unmarshal")
+	}
+
+	policiesMap := make(map[string]string)
+	for i := 0; i < len(policies); i += 1 {
+		policiesMap[policies[i].Name] = policies[i].Id
+	}
+	return policiesMap, nil
+}
+
+func (e *EnvironmentManager) createRBAC(envs []Environment, policies map[string]string) error {
+	for _, env := range envs {
+		if _, policyExists := policies[env.App.ID]; policyExists {
+			// Policy already exists, not recreating
+			continue
+		}
+		policyDefinition := &PolicyDefinition{
+			V1: PolicyDefinitionV1{
+				Name: "Policy Name",
+				Resources: PolicyResourcesV1{
+					Allowed: []string{fmt.Sprintf("kots/app/%s/**", env.App.ID), "kots/license/**"},
```

Comment:
Should we add `team/serviceaccount/create` and `team/serviceaccount/list` to the policy? This will allow the end user to create a service account token, copy it to their clipboard and see it in a list. They would see the list of al team SA token but will not be able to delete any of them including their own.